scripts/github-status/github-status.rb

```diff
@@ -142,7 +146,7 @@ def show_forcerebuild_pull_requests
 
 optparse.parse!
 
-ghc=GHClientHandler.new()
+ghc=GHClientHandler.new(config = { repository: options[:repository] })
```

Comment:
Really don't mind, just thought I'd mention as it's maybe not a well-known trick in Ruby and can sometimes make code a bit more concise and clean :)